doc/how_to/index.md

```diff
@@ -225,6 +225,25 @@ How to run Panel applications entirely in the browser using WebAssembly (Wasm),
 
 ::::
 
+## Migrate to Panel
```

Comment:
I have no way of validating this is correct

Reply:
Yes. It's those Sphinx restricted text special blocks. I'm not able to build the docs. So for example validation the grid, tic tree etc is valid, is simply not possible for me.